packages/blocks/src/api/test/registration.js

```diff
@@ -1625,6 +1625,39 @@ describe( 'blocks', () => {
 				'Block bindings source "core/test-source" is already registered.'
 			);
 		} );
+
+		it( 'should not be overriden if bootstrap happens after registration', () => {
```

Comment:
The test description should explain better the behavior. How about:

```suggestion
		it( 'should correctly merge properties when bootstrap happens after registration', () => {
```